src/components/Currency.js

```diff
@@ -32,7 +32,7 @@ const Currency = ({ abbreviate, currency, precision, value, ...styles }) => (
 
 Currency.propTypes = {
   /** The amount to display, in cents */
-  value: PropTypes.number.isRequired,
```

Comment:
Doesn't make sense to not pass a number to this component. What do you think?

Reply:
I agreed but during the tests, I catch some component not passing the value.

Reply:
Would be better to keep it isRequired and fix these occurences, but that can be handled later as a follow up.

Reply:
Ok.